Nova landing/agrupador de Split de Pagamentos (#2243)
mercadopago/devsite-docs: guides/split-payment/configuration.en.md

# Create configuration

To configure the integration with the Split Payments solution, you will need to [create your application](#bookmark_create_application), [request permissions from your users](#bookmark_request_permission_from_users), and [obtain the credentials](#bookmark_obtain_credentials). Keep reading to create the required configuration.

## Create application

Create your application to integrate with the Split Payments solution by following the steps below.

   > NOTE
   >
   > Important
   >
   > During the creation of your application, it may be necessary to reauthenticate your identity. If you have already completed the verification, reauthentication will be requested. Otherwise, you will be redirected to submit the necessary documents. This additional step is essential to protect your account and ensure compliance with operations. You can refer to the [Developer dashboard documentation](/developers/es/docs/split-payment/additional-content/your-integrations/dashboard) if you have any questions on how to use it.

> 1. Access [Your Integrations](https://www.mercadopago.com.br/developers/panel/app). Once there, click on the **Create application** button, located in the upper right corner.

Reviewer: ```suggestion
1. Access [Your Integrations](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/panel/app). Once there, click on the **Create application** button, located in the upper right corner.
```

2. Enter a name to identify your application (you have a limit of 50 characters).
3. Choose the **Online Payments** solution.

> 4. When choosing the product to integrate, you can select "Checkout Pro" or "Checkout API."

Reviewer: ```suggestion
4. When choosing the product to integrate, you can select "Checkout Pro" or "Checkout API".
```

5. Choose the **Marketplace** integration model.
6. Once you have completed the requested information, click on **Create application** and you're done!

### Configure the Redirect URL

In this step, it is necessary to configure the Redirect URL so that sellers can authorize the marketplace to carry out sales.

> After creating the application, it is necessary to go to the editing screen to fill in the Redirect URL field (in OAuth requests, it is displayed as redirect_uri), which must contain the URL of the marketplace site where the seller's token will be sent upon completing the linking process.

Reviewer: ```suggestion
After creating the application, it is necessary to go to the editing screen to fill in the Redirect URL field (in OAuth requests, it is displayed as `redirect_uril`, which must contain the URL of the marketplace site where the seller's token will be sent upon completing the linking process.
```

Reviewer 2: @pcanosa ! ojo que quedó mal el valor y faltó cerrar el paréntesis. Sería: (in OAuth requests, it is displayed as `redirect_uri`)

Reviewer: Gracias por notarlo, Gosti. Ahí lo acomodé, aunque sería "redirect_url"

Author: Oi, gente! Nesse caso, seria redirect_uri mesmo (Uniform Resource Identifier).

![Redirect URL](/images/split-payment/redirect-url-es.png)

## Request permission from users

To request permission from your users to manage sales on their behalf, it is necessary to implement the authorization flow using OAuth. Follow the steps below:

1. Redirect your users to the following URL to authorize sales management:

| Value                | Description                                                                                                          |
|----------------------|----------------------------------------------------------------------------------------------------------------------|
| `<APP_ID>`           | Value obtained during [application creation](/developers/en/docs/split-payment/integration-configuration/create-application).    |

> | `<REDIRECT_URI>`     | Value entered in the Redirect URI field during [Redirect URL configuration](/developers/en/docs/split-payment/integration-configuration/create-application). |

Reviewer: ```suggestion
| `<REDIRECT_URL>`     | Value entered in the Redirect URL field during [Redirect URL configuration](/developers/en/docs/split-payment/integration-configuration/create-application). |
```

```curl
http://<REDIRECT_URI>?code=AUTHORIZATION_CODE
```

> NOTE
>
> Nota
>
> The `AUTHORIZATION_CODE` is valid for 10 minutes and will be used to create the necessary credentials.

## Obtain credentials

Use the authorization code obtained in the previous step to acquire user credentials through the [OAuth API](/developers/en/reference/oauth/_oauth_token/post), enabling you to manage their sales.

| Parameter                | Description                                                                                      |
|--------------------------|--------------------------------------------------------------------------------------------------|
| `<CLIENT_ID>`            | Value of APP_ID obtained in the details of your application.                                      |
| `<CLIENT_SECRET>`        | Your SECRET_KEY, also available in the details of your application.                              |
| `<AUTHORIZATION_CODE>`   | Authorization code obtained when redirecting the user back to your site.                     |

> | `<REDIRECT_URI>`         | Should be the same Redirect URI configured in your application.                                     |

Reviewer: ```suggestion
| `<REDIRECT_URL>`         | Should be the same Redirect URL configured in your application.                                     |
```
